src/components/classComp/movie-card/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './style.less'
+import {Movie} from '../../../models/movieDay';
+
+
+interface State {
+}
+
+interface Props {
+    info: Movie;
+ }
+
+
+class MovieCard extends React.Component<Props, State> {
+
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+        };
+    }
+
+    formatTime(time: Date) {
+        let date = new Date(time);
+        let movieTime = '';
+        let hours = date.getHours();
+        let minutes = date.getMinutes();
+        if(hours < 10 && minutes < 10) {
+            movieTime = `0${hours}:0${minutes} `;
```

Comment:
Could be rewritten to a shorter expression if 'minutes' and 'hours' are treated separately and then combined into one.

```suggestion
        hours = hours < 10 ? hours : `0${hours}`
        minutes = minutes < 10 ? minutes : `0${minutes}`
        const movieTime = `${hours}:${minutes}`
```